pages/docs/open-source-roadmap/index.js

```diff
@@ -169,7 +169,7 @@ export default function DocsIndex() {
                                 <div className="text-sm">
                                   <a
                                     href={roadmapItem.action.href}
-                                    className="font-medium text-primary-high dark:text-primary-low"
+                                    className="font-medium text-primary-high dark:text-primary-low  hover:text-tertiary-medium dark:hover:text-tertiary-medium"
```

Comment:
I think it would be better to remove the classes and use the `Link` component, this will help us keep consistency within the project for links